test/functional/prioritise_transaction.py

```diff
@@ -109,13 +109,9 @@ def run_test(self):
         tx_hex = self.nodes[0].signrawtransaction(raw_tx)["hex"]
         tx_id = self.nodes[0].decoderawtransaction(tx_hex)["txid"]
 
-        try:
-            self.nodes[0].sendrawtransaction(tx_hex)
-        except JSONRPCException as exp:
-            assert_equal(exp.error['code'], -26) # insufficient fee
-            assert(tx_id not in self.nodes[0].getrawmempool())
-        else:
-            assert(False)
+        # This will raise an exception due to min relay fee not being met
+        assert_raises_jsonrpc(-26, "66: insufficient priority", self.nodes[0].sendrawtransaction, tx2_hex)
```

Comment:
```suggestion
        assert_raises_jsonrpc(-26, "66: min relay fee not met", self.nodes[0].sendrawtransaction, tx2_hex)
```

see https://github.com/dashpay/dash/pull/2763#discussion_r265353260

Reply:
https://github.com/dashpay/dash/pull/2763/commits/0b8012bd55afc6cc87e81a4f5c1205fe6b8394dc included here. Rebased and squashed